packages/app-content-pages/server/server.js

```diff
@@ -27,13 +28,20 @@ const handle = app.getRequestHandler()
 app.prepare().then(() => {
   const server = express()
 
+  server.use(Sentry.Handlers.requestHandler())
   setLogging(server)
 
   server.get('*', (req, res) => {
     setCacheHeaders(req, res)
     return handle(req, res)
   })
 
+  server.use(Sentry.Handlers.errorHandler())
+  server.use(function onError(error, req, res, next) {
+    res.statusCode = 500
```

Comment:
[minor] I was going to ask, "are all errors going to be 500s? Would this override 400-range errors?" but then I realised this pattern is _literally_ what's documented on the Sentry Express readme, doy.